fe/fe-core/src/main/java/com/starrocks/analysis/ArithmeticExpr.java

```diff
@@ -325,8 +325,7 @@ private void rewriteNonDecimalOperation() throws AnalysisException {
                 break;
             case DIVIDE:
                 commonType = getCommonType(t1, t2);
-                if (commonType.getPrimitiveType() == PrimitiveType.BIGINT
-                        || commonType.getPrimitiveType() == PrimitiveType.LARGEINT) {
+                if (commonType.getPrimitiveType().isIntegerType()) {
```

Comment:
Please make clear the title and commit message.

Reply:
I've made them clearer.